_posts/2016-01-29-back-to-business.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Back to Business
+author: riikkanen
+excerpt: Turning from a mother to a software developer. A normal day for me.
+categories: 
+- normipäivä
+- programming
+- Java
+- Solita
+---
+
+##From a Mother to a Software Developer
+
+Clock alarms at 6.15 am. Today I'm lucky and also the children have slept so long. Today is working day, one of my three working days per week. On other days I take care of my three children at home, which is a convenient arrangement for the time being and enabled by my employee, Solita.
+
+After waking up, eating porridge, feeding porridge for my 1,5 years old daughter, making braids for my 6 years old daughter and saying bye-byes also to my 4 years old son and my dear husband, I take my bicycle and pedal to work. Cycling is a wonderful way to switch the context in my head from home matters to work issues. When I arrive to our office, I take my bag and change clothes, like Superman in a phone booth, turning from a mother to a software developer. 
+
+This is my 12th working day after quite a long break, a little over 1,5 years maternity leave. Before that I had been working in KIRRE project since 2010. In KIRRE I worked as a software developer and had my hands on both in back end and front end. KIRRE project was dealing with registry of legal confirmations to real properties. 
+
+##Jump to the Moving Train
+
+After working a long time in the same project, it seemed to be reasonable to change the project, from real properties to a completely different world, railways. LIIKE project has been started in 2008 in co-operation with Liikennevirasto. LIIKE projects deals with managing the capacity of railways and supervising and controlling the actual traffic. The project contains maintenance and support, but also further development.
+
+It's always a challenge to familiarize oneself with a new business branch, ecspecially when the project has already been active for a long time and the business logic is full of little but still essential details. A huge amount of information is just waiting to be digested.
```

Comment:
_especially_ when _a_ project

and _its_ business logic
little but still essential -> small yet essential